internal/cli/tables-to-go-cli.go

```diff
@@ -45,18 +46,32 @@ func Run(settings *settings.Settings, db database.Database, out output.Writer) (
 		}
 
 		if err = db.GetColumnsOfTable(table); err != nil {
-			return fmt.Errorf("could not get columns of table %s: %v", table.Name, err)
+			if !settings.Force {
+				return fmt.Errorf("could not get columns of table %s: %v", table.Name, err)
+			}
+			fmt.Printf("could not get columns of table %s: %v\n", table.Name, err)
```

Comment:
Lets use %q here for table name format string - applies also for all other error messages.

Reply:
resolved in 396b9e869b3